test/org/zaproxy/zap/extension/pscanrulesAlpha/PiiScannerUnitTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2017 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+/**
+ * @author Michael Kruglos (@michaelkruglos)
+ */
+@RunWith(Parameterized.class)
+public class PiiScannerUnitTest extends PassiveScannerTest {
```

Comment:
Probably better to rename it to `PiiScannerCreditCardUnitTest`(?), as this is specific to CC.

It would be great to have a test with multiple alerts in the same message.